README.md

```diff
@@ -1,5 +1,15 @@
 # Timesheetsdb
-Timesheets web application
+Time sheets database for web application
+
```

Comment:
Give a general description of the overall application, and point to the UI project as well (with hyperlink).
You could copy the diagram from the slides I made.

Say how it exposes REST web services on port 3000.

Also give a section on the Ansible install too (with hyperlink) - I know it's a private repo, but we want our own people to be able to find it.